samples/matter/lock/Kconfig

```diff
@@ -58,6 +58,12 @@ config APPLICATION_OTHER_LABEL
 	default "Wi-Fi" if NET_L2_OPENTHREAD
 	default "Thread" if CHIP_WIFI
 
+config THREAD_WIFI_SWITCHING_CLI_SUPPORT
+	bool "Add CLI command for triggering of the switch between Thread and Wi-Fi"
+	default n
+	depends on THREAD_WIFI_SWITCHING
```

Comment:
fyi: `depends on` is not needed if this is already in the `if` scope.

Reply:
Yes, initially the whole config definition was placed outside the if case.